pages/api/index.ts

```diff
@@ -8,5 +8,5 @@ export default async function handler(
 ) {
   response
     .status(200)
-    .json({ sessions: `http://${request.headers.host}/api/sessions` });
+    .json({ sessions: `https://${request.headers.host}/api/sessions` });
```

Comment:
nice, this is a good change - I guess there's no more `http:` occurrences anywhere?

Reply:
i think this must be reverted since localhost doesn't work on https in next.js

Reply:
ah ok yeah makes sense